docs/docs/creating-and-modifying-pages.md

```diff
@@ -263,6 +263,23 @@ export default Page
 
 Page context is serialized before being passed to pages. This means it can't be used to pass functions into components and `Date` objects will be serialized into strings.
 
+### Optimizing pages for content sync
+
+If using the Preview UI Content Sync feature on Gatsby Cloud, an optional parameter, `ownerNodeId` can be passed to the `createPage` invocation to give you greater control over how content is previewed. By passing a value to `ownerNodeId`, you can ensure that the CMS content is properly mapped to the page you would like to preview it on. For example, the value of `ownerNodeId` can be set to `node.id` when pages are generated for each queried node.
+
+```javascript:title=gatsby-node.js
+const posts = result.data.allContentfulBlogPost.nodes
+
+posts.forEach((post, index) => {
```

Comment:
Super nit and non-blocking, but not sure it's worth including the `index` param in this if it's not utilized in the example.